examples/hardhat/hardhat.config.ts

```diff
@@ -16,21 +16,21 @@ const config: HardhatUserConfig = {
     },
   },
   networks: {
-    'sapphire_mainnet': {
+    'sapphire': {
```

Comment:
I'm a little conflicted here, but 🤷‍♂️? 

```suggestion
    sapphire: {
```